lib/fungus_toast_web/controllers/game_controller.ex

```diff
@@ -5,18 +5,35 @@ defmodule FungusToastWeb.GameController do
 
   action_fallback FungusToastWeb.FallbackController
 
+  def index(conn, params) do
+    user_id = Map.get(params, "user_id")
+    active = Map.get(params, "active", "false") |> active?()
```

Comment:
Kinda of a nitpick, and no real reason to change this, but I think we could delete lines 34-38 if we changed this to 
`active = Map.get(params, "active") in ["1", "true"] ` couldn't we?

Reply:
Didn't know that was a thing. We'd no longer return an error, but I think that would be fine